example/lib/main.dart

```diff
@@ -54,19 +54,15 @@ class NodeEditorExampleScreenState extends State<NodeEditorExampleScreen> {
 
     _nodeEditorController = FlNodeEditorController(
       projectSaver: (jsonData) async {
-        if (kIsWeb) return false;
-
         final String? outputPath = await FilePicker.platform.saveFile(
           dialogTitle: 'Save Project',
           fileName: 'node_project.json',
           type: FileType.custom,
           allowedExtensions: ['json'],
+          bytes: utf8.encode(jsonEncode(jsonData)),
         );
 
-        if (outputPath != null) {
-          final File file = File(outputPath);
-          await file.writeAsString(jsonEncode(jsonData));
-
+        if (outputPath != null || kIsWeb) {
```

Comment:
On web `saveFile` will always return `null` no matter what, as return value is a path to saved file, and browser will not expose that information to us